config/v1/types_infrastructure.go

```diff
@@ -116,6 +116,9 @@ const (
 
 	// KubevirtPlatformType represents KubeVirt/Openshift Virtualization infrastructure.
 	KubevirtPlatformType PlatformType = "KubeVirt"
+
+	// EquinixMetalPlatformType represents Equinix Metal infrastructure.
+	EquinixMetalPlatformType PlatformType = "EquinixMetal"
```

Comment:
does the `+kubebuilder:validation` comment on the top of this structure need to be regenerated to include `EquinixMetal`?

Reply:
yes

Reply:
Updated, thanks!